### ISSUE TITLE
remove unnecessary output network of KO

### ISSUE THREAD

**Issue by jamesjcai** [cailab-tamu/scTenifoldKnk#9]
outputList$tensorNetworks$KO <- Matrix(KO)
Is this necessary? Matrix(KO) and Matrix(WT) only differ by one row, but will take same amount of disk/memory space.

**Comment by dosorio** [comment]
Yes, it can be removed. The final matrix is smaller than 200 MB in general because it is encoded as an sparse matrix.
